<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/ionicons-2.0.1/css/ionicons.min.css" rel="stylesheet" />
<link href="https://fonts.googleapis.com/css?family=Nunito+Sans|Roboto|Lobster" rel="stylesheet">

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 64px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h2 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h3 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h4 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h5 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h6 {
  padding-top: 69px;
  margin-top: -69px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">



<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">
      	<img alt="Nemabiome Logo" src="images/logo/nemabiome_compact_64.png" style="width:35px;height:35px">
      </a>
    </div>
    	
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
				
				<li>
					<a href="overview.html">
						<span class="ion ion-map"></span>
    				Overview
					</a>
				</li>
	
			<!-- Parasite -->
				<li>
  				<a href="parasite.html">
    				<span class="ion ion-bug"></span>
    				Parasite Prep
     			</a>
				</li>
				
			<!-- Sequencing -->	
				<li>
  				<a href="sequencing.html">
    				<span class="ion ion-erlenmeyer-flask"></span>
    				Sequencing
     			</a>
				</li>
				
			<!-- Analysis -->
				<li>
  				<a href="analysis.html">
    				<span class="ion ion-monitor"></span>
    				Analysis
     			</a>
				</li>
			
			<!-- Interpretation -->
				<li>
  				<a href="interpretation.html">
    				<span class="ion ion-stats-bars"></span>
    				Interpretation
     			</a>
				</li>

			</ul>
			
			<!-- Citation and about -->
	    <ul class="nav navbar-nav navbar-right">

				<li>
  				<a href="citation.html">
    				<span class="ion ion-information-circled"></span>
    		    Citation
  				</a>
				</li>
				
			
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<div id="analysis-protocols" class="section level1 tabset tabset-pills tabset-fade">
<h1>Analysis Protocols</h1>
<p>Once you have generated the data from the MiSeq you must complete the following steps in order to analyze the data.</p>
<div id="software-requirements" class="section level2">
<h2>Software Requirements</h2>
<p>The primary software used for analysis is Mothur. This program is command line based, and will require you to type simple commands into either Terminal (Mac) or Command Prompt (Windows).</p>
<p>To download Mothur go to the Mothur github <a href="https://github.com/mothur/mothur/releases/latest">downloads</a> page and download the latest version for your operating system. Unzip then move this folder to your home directory (your C: Drive on a PC, or your ‘User’ folder on a Mac).</p>
<div class="panel panel-info">
<div class="panel-heading">
Tip
</div>
<div class="panel-body">
<p>To check that you have placed the folder in the correct location, open up either Terminal (Mac) or Command Prompt (PC). Type <code>~/mothur/mothur</code> (Mac) or <code>C:\mothur\mothur.exe</code> (PC). If your screen looks similar to this:</p>
<div class="figure">
<img src="images/mothur_screenshot.png" />

</div>
<p>You have done it correctly.</p>
</div>
</div>
</div>
<div id="preparing-files-for-analysis" class="section level2">
<h2>Preparing files for analysis</h2>
<div id="downloading-sequencing-data-files-from-basespace" class="section level3">
<h3>Downloading sequencing data files from BaseSpace</h3>
<p>To download the sequencing fields from BaseSpace. Go to <a href="https://basespace.illumina.com/home/index" class="uri">https://basespace.illumina.com/home/index</a>, log-in, and navigate to your sequencing run. Click “Download” and select “All FASTQ files for this run”. Then click “Download your files”. BaseSpace uses the “BaseSpace Downloader” to download your files. If you have not previously downloaded this program, you must download it at this time. Select the location to download your files.</p>
<p>Once your files have downloaded, navigate to the folder where the files were downloaded. There will be one folder that corresponds to each sample that was sequenced. In each of these folders there will be two files, one that corresponds to the forward reads, and one that corresponds to the reverse reads. The forward and reverse read files for all samples will need to be moved to a single folder for analysis to conduct the analysis.</p>
<div class="panel panel-success">
<div class="panel-heading">
Recommendation
</div>
<div class="panel-body">
<p>Create a folder on your home directory with the name of your sequencing run (for the rest of this example it will be called “MiSeq_Run”). Move all of your files into this folder.</p>
</div>
</div>
<p>Once the files have been moved, the files must be unzipped/uncompressed. This can be done with the Archive Utility on a Mac and any file compression software, such as WinZip, on a PC. Make sure you unzip your files into the folder that you have created (i.e. “Miseq_Run”)</p>
</div>
<div id="create-sample-list" class="section level3">
<h3>Create sample list</h3>
<p>You will need to create a file that contains the name of your sample, and specifies which files correspond to this sample. This will tell the program which files to analyze for each particular sample. You can download the file “nemabiome.files” in the <a href="interpretation.html">Examples</a> section for an example of what this file must look like. Each line in this file represents a single sample that must be analyzed. The first column is the sample name, the second column is the file name for the forward read file, and the third column is the name of the reverse read file. Each column is separated by a single tab. An example is also listed below.</p>
<p>Example:</p>
<pre><code>sample1 sample1_S1_L001_R1_001.fastq     sample1_S1_L001_R2_001.fastq
sample2 sample2_S2_L001_R1_001.fastq     sample2_S2_L001_R2_001.fastq
sample3 sample3_S3_L001_R1_001.fastq     sample3_S3_L001_R2_001.fastq
...</code></pre>
<p>Sample names may vary between different Illumina platforms, so please check the names of your downloaded files, to ensure the file names are correct. Please name this file as “stability.files”, with the “.files” file extension. Place this file in your folder that you created for analysis. This is the only file that you will need to create from scratch for this analysis, if you use the provided files.</p>
</div>
<div id="download-analysis-files" class="section level3">
<h3>Download analysis files</h3>
<p>Download the analysis files <a href="downloads/nematode_ITS2_database_version1_3.fasta" download>Nematode ITS2 database version 1.3</a>, <a href="downloads/nematode_taxonomy_1_3.tax" download>Nematode taxonomy file (version 1.3)</a> and <a href="downloads/batchfile.txt" download>Mothur batch file</a>. Place these files in the folder that contains your sequencing data (i.e. ‘MiSeq_Run’)</p>
<p>For a standard application do not make any changes to these files. If you are aware of the number of processors that your computer has, open up the <code>batchfile.txt</code> file, go to the first line and change the parameter “processors” from “processors=2” to the number of processors in your computer.</p>
</div>
</div>
<div id="running-analysis" class="section level2">
<h2>Running analysis</h2>
<p>Once all files have been downloaded, open up Terminal or Command Prompt. You will then need to navigate to the folder you created, which can be done using the “cd” command. If you created your analysis folder in your home directory type:</p>
<pre><code>cd MiSeq_Run</code></pre>
<p>This will change to this folder.</p>
<p>Then type:</p>
<pre><code>~/mothur/mothur batchfile.txt</code></pre>
<p>(On a Mac)</p>
<p>OR</p>
<pre><code>C:\mothur\mothur.exe C:\MiSeq_Run\batchfile.txt</code></pre>
<p>(On a PC)</p>
<p>This will launch Mothur, and begin to complete the commands that are contained within the batchfile. You will need to leave your computer open and running while the analysis completes. The amount of time that it will take to run will vary with your computer, and the number of samples/reads that you will be analyzing. General run times will probably be around ~4 hours, on a standard computer, 96 samples with an average read depth of ~50,000 reads.</p>
<p>Once the analysis has completed, you will have a file called <code>nemabiome_results.summary</code>. You can open this file up and copy the contents into an Excel workbook to properly view the data.</p>
</div>
<div id="dada2-example-workflow" class="section level2">
<h2>DADA2 Example Workflow</h2>
<p>The following is a sample analysis using the DADA2 pipeline (<a href="https://benjjneb.github.io/dada2/tutorial.html" class="uri">https://benjjneb.github.io/dada2/tutorial.html</a>). This is meant to be an example and not a definitive workflow. First we load packages and set a seed for reproducibility.</p>
<pre><code>library(DECIPHER)
packageVersion(&quot;DECIPHER&quot;)

library(dada2)
packageVersion(&quot;dada2&quot;) 

library(ShortRead)
packageVersion(&quot;ShortRead&quot;) 

library(Biostrings)
packageVersion(&quot;Biostrings&quot;)

set.seed(106)</code></pre>
<p>To use the DADA2 pipeline, primers must be removed from amplicons. This can be done in R using cutadapt. Here we specify the path of sequences to be processed (using “/Miseq_Run” from previous examples) and the path of Cutadapt on your machine (we used “/home/gilleardlab/anaconda3/bin/cutadapt”). You will also need to specify the primers you used as the “FWD” and “REV” variables. We start off by creating all possible orientations of the forward and reverse sequences.</p>
<pre><code>path&lt;- &quot;Miseq_Run&quot;
list.files(path)

fnFs&lt;-sort(list.files(path, pattern = &quot;_R1.fastq.gz&quot;, full.names = TRUE)) 
fnRs &lt;-sort(list.files(path, pattern = &quot;_R2.fastq.gz&quot;, full.names = TRUE))


#identifying the primers
FWD&lt;-&quot;ACGTCTGGTTCAGGGTTGTT&quot;
REV &lt;- &quot;TTAGTTTCTTTTCCTCCGCT&quot;

allOrients&lt;-function(primer) {   #creates all orientations of the primer sequence)
  require(Biostrings)
  dna&lt;-DNAString(primer) #biostring works w dnastring objects
  orients&lt;-c(Forward=dna, Complement=complement(dna), Reverse=reverse(dna),
             RevComp=reverseComplement(dna))
  return(sapply(orients, toString)) #now converting back into character vector
}
FWD.orients&lt;-allOrients(FWD)
REV.orients&lt;-allOrients(REV)

FWD.orients</code></pre>
<p>Next we will remove all sequences containing ambiguous bases, and count the number of times our primers are observed in the remaining sequences.</p>
<pre><code>#fltering the sequences
fnFs.filtN &lt;- file.path(path, &quot;filtN&quot;, basename(fnFs)) 
fnRs.filtN &lt;- file.path(path, &quot;filtN&quot;, basename(fnRs))
filterAndTrim(fnFs, fnFs.filtN, fnRs, fnRs.filtN, maxN = 0, multithread = TRUE)

#counting number of primers 
primerHits &lt;- function(primer, fn) {
  # Counts number of reads in which the primer is found
  nhits &lt;- vcountPattern(primer, sread(readFastq(fn)), fixed = FALSE)
  return(sum(nhits &gt; 0))
}
rbind(FWD.ForwardReads = sapply(FWD.orients, primerHits, fn = fnFs.filtN[[1]]), 
      FWD.ReverseReads = sapply(FWD.orients, primerHits, fn = fnRs.filtN[[1]]), 
      REV.ForwardReads = sapply(REV.orients, primerHits, fn = fnFs.filtN[[1]]), 
      REV.ReverseReads = sapply(REV.orients, primerHits, fn = fnRs.filtN[[1]]))</code></pre>
<p>Now we can use cutadapt to remove primers, and check that they have been successfully removed.</p>
<pre><code>cutadapt &lt;- &quot;/home/gilleardlab/anaconda3/bin/cutadapt&quot; # CHANGE ME to the cutadapt path on your machine
system2(cutadapt, args = &quot;--version&quot;) # Run shell commands from R

#now creating output filenames for cutadapted files, and define parameters to give cutadapt command
#critical parameters are the primers and they need to be in the right orientation

path.cut &lt;- file.path(path, &quot;cutadapt&quot;)
if(!dir.exists(path.cut)) dir.create(path.cut)
fnFs.cut &lt;- file.path(path.cut, basename(fnFs))
fnRs.cut &lt;- file.path(path.cut, basename(fnRs))  #cutadapt not working -&gt; due to files not gz?

FWD.RC &lt;- dada2:::rc(FWD)
REV.RC &lt;- dada2:::rc(REV)
# Trim FWD and the reverse-complement of REV off of R1 (forward reads)
R1.flags &lt;- paste(&quot;-g&quot;, FWD, &quot;-a&quot;, REV.RC) 
# Trim REV and the reverse-complement of FWD off of R2 (reverse reads)
R2.flags &lt;- paste(&quot;-G&quot;, REV, &quot;-A&quot;, FWD.RC) 
# Run Cutadapt
for(i in seq_along(fnFs)) {
  system2(cutadapt, args = c(R1.flags, R2.flags, &quot;-n&quot;, 2, # -n 2 required to remove FWD and REV from reads
                             &quot;-o&quot;, fnFs.cut[i], &quot;-p&quot;, fnRs.cut[i], # output files
                             fnFs.filtN[i], fnRs.filtN[i])) # input files
}

# remember to check things - can check presence of primers in the first cutadapted sample 
rbind(FWD.ForwardReads = sapply(FWD.orients, primerHits, fn = fnFs.cut[[1]]), 
      FWD.ReverseReads = sapply(FWD.orients, primerHits, fn = fnRs.cut[[1]]), 
      REV.ForwardReads = sapply(REV.orients, primerHits, fn = fnFs.cut[[1]]), 
      REV.ReverseReads = sapply(REV.orients, primerHits, fn = fnRs.cut[[1]]))
</code></pre>
<p>Now that primers are removed, we can begin the DADA2 pipeline. We start with plotting quality profiles of our sequences to determine suitable quality filtering parameters.</p>
<pre><code>cutFs &lt;- sort(list.files(path.cut, pattern = &quot;_R1.fastq.gz&quot;, full.names = TRUE)) #remember to match file names 
cutRs &lt;- sort(list.files(path.cut, pattern = &quot;_R2.fastq.gz&quot;, full.names = TRUE))

# Extract sample names, assuming filenames have same format, may need to alter!!!
get.sample.name &lt;- function(fname) strsplit(basename(fname), &quot;_R&quot;)[[1]][1]
sample.names &lt;- unname(sapply(cutFs, get.sample.name))
sample.names

#plotQualityProfile(cutFs[1:2])+ggtitle(&quot;Forward&quot;)
#plotQualityProfile(cutRs[1:2])+ggtitle(&quot;Reverse&quot;)</code></pre>
<p>For the purposes of our analysis, we chose to conduct quality filtering with maximum expected errors of 2 for the forward sequence, 5 for the reverse, to truncate after a quality score of 2 or lower, and to filter out amplicons with lengths shorter than 50 base pairs. Again, filters for your own pipeline may differ. After quality filtering, we can run our sequences through learnErrors to create an error model, and then dereplicate our sequences.</p>
<pre><code>#filtering reads 
filtFs&lt;-file.path(path.cut, &quot;filtered&quot;, basename(cutFs))
filtRs&lt;-file.path(path.cut, &quot;filtered&quot;, basename(cutRs))

out &lt;- filterAndTrim(cutFs, filtFs, cutRs, filtRs, maxN = 0, maxEE = c(2, 5), 
                     truncQ = 2, minLen = 50, rm.phix = TRUE, compress = TRUE, multithread = TRUE)  
head(out)
out

#learning errors
errF&lt;-learnErrors(filtFs, multithread=TRUE)
errR&lt;-learnErrors(filtRs, multithread=TRUE)

#visualize the error rates w code below so you can see
# plotErrors(errF, nominalQ=TRUE)

# denoising data 
derepFs&lt;-derepFastq(filtFs, verbose = TRUE)
derepRs&lt;-derepFastq(filtRs, verbose=TRUE)</code></pre>
<p>After dereplication, we can denoise our sequences using the DADA algorithm. We will also merge our paired-end sequences, allowing for up to 1 mismatch in the overlap region and returning all sequences whether they merged or not.</p>
<pre><code>dadaFs&lt;-dada(derepFs, err=errF, multithread = TRUE)
dadaRs&lt;-dada(derepRs, err=errR, multithread=TRUE)

mergers&lt;-mergePairs(dadaFs, derepFs, dadaRs, derepRs, verbose=TRUE, maxMismatch = 1, returnRejects = TRUE)</code></pre>
<p>Before we can assign taxonomy, we can construct a sequence table, remove chimeras using the removeBimeraDenovo command and then create a table of how many sequences were retained after each processing step.</p>
<pre><code>seqtab&lt;-makeSequenceTable(mergers)
dim(seqtab) 

#Remove chimeras 
seqtab.nochim&lt;-removeBimeraDenovo(seqtab, method=&quot;consensus&quot;, multithread=TRUE, verbose=TRUE)
#inspect distribution sequence lengths
table(nchar(getSequences(seqtab.nochim)))

#track reads through the pipeline - inspect the number of reads that made it through each step in the pipeline -- IMPORTANT

getN &lt;- function(x) sum(getUniques(x))
track &lt;- cbind(out, sapply(dadaFs, getN), sapply(dadaRs, getN), sapply(mergers, 
                                                                       getN), rowSums(seqtab.nochim))

# If processing a single sample, remove the sapply calls: e.g. replace
# sapply(dadaFs, getN) with getN(dadaFs)
colnames(track) &lt;- c(&quot;input&quot;, &quot;filtered&quot;, &quot;denoisedF&quot;, &quot;denoisedR&quot;, &quot;merged&quot;, 
                     &quot;nonchim&quot;)
rownames(track) &lt;- sample.names     #what&#39;s considered to be an overlarge drop?? - reads seem fine
head(track)</code></pre>
</div>
<div id="downloads" class="section level2">
<h2>Download list</h2>
<ul>
<li><a href="downloads/batchfile.txt" download="downloads/batchfile.txt">Mothur batch file</a></li>
<li><a href="downloads/nematode_taxonomy_1_3.tax" download>Nematode taxonomy file (version 1.3)</a></li>
<li><a href="downloads/nematode_ITS2_database_version1_3.fasta" download>Nematode ITS2 database version 1.3</a></li>
</ul>
</div>
</div>

   </div> <!-- articleBandContent -->
</div> <!-- pageContent -->
 

<footer class="footer">
	<nav class="navbar navbar-default navbar-fixed-bottom">
	  <div class="container">
	    <div class="col-sm-12 text-center navbar-text">
	        <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">
    				<img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" />
    		</a>
    		This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
	    </div>
	  </div>
	</nav>
</footer>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
